AppCenterCrashes/AppCenterCrashes/Model/MSACWrapperExceptionModel.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+#ifndef MSAC_WRAPPER_EXCEPTION_MODEL_H
+#define MSAC_WRAPPER_EXCEPTION_MODEL_H
+
+#if __has_include(<AppCenterCrashes/MSACExceptionModel.h>)
+#import <AppCenterCrashes/MSACExceptionModel.h>
+#import <AppCenterCrashes/MSACWrapperExceptionModel.h>
+#else
+#import "MSACExceptionModel.h"
+#import "MSACWrapperExceptionModel.h"
+#endif
+
+#if __has_include(<AppCenter/MSACSerializableObject.h>)
```

Comment:
One `#if` should be enough. Just put these imports in first condition.

Reply:
no, these imports from different modules (core and crashes)